modules/api/php/endpoints/candidate/visit/image/format/downloadfile.class.inc

```diff
@@ -100,23 +126,22 @@ class Thumbnail extends Endpoint implements \LORIS\Middleware\ETagCalculator
      */
     private function _handleGET(): ResponseInterface
     {
-        $info = $this->_image->getThumbnailFileInfo();
-
-        if (!$info->isFile()) {
-            error_log('Thumbnail not found');
+        if (!$this->_info->isFile()) {
+            $this->logger->notice("$this->_file_type file not found");
             return new \LORIS\Http\Response\JSON\NotFound();
         }
 
-        if (!$info->isReadable()) {
-            error_log('Thumbnail exists but is not readable by webserver');
+        if (!$this->_info->isReadable()) {
+            $this->logger->notice(
```

Comment:
This should probably be logged at the level of "error" log level.

See the different log level definitions in PSR-3. https://www.php-fig.org/psr/psr-3/

The database referencing a file that the server can't read seems more severe than a notice.